src/conditionals/programs_with_decision_points/hard/q10/DinnerCost.java

```diff
@@ -0,0 +1,44 @@
+package conditionals.programs_with_decision_points.hard.q10;
+
+/*
+ * Write a Java Program that determines the cost of a dinner considering if the service was good i.e. if the
```

Comment:
This does not make sense

Reply:
This should be changed to something like: 
`Write a Java program that determines the cost of dining at a restaurant depending on the quality of service. The program should prompt the user about the cost of the food and if the service is good. If the service was good, then the user tips 20%. If the service was okay, then the user only tips 15%. The cost of dinner should be round to the nearest integer. If the cost of food is less than 0, the program should print "Invalid Input!" to the user. 
`